install.hs

```diff
@@ -449,18 +474,21 @@ getStackGhcPath ghcVersion = do
   Stdout ghc <- execStackWithYaml ghcVersion ["path", "--compiler-exe"]
   return $ trim ghc
 
+getStackGhcPathShake :: Action GhcPath
```

Comment:
why the `Shake` suffix? 
Just as a question.

Reply:
Uses the `shake.yaml` instead of `stack-*.yaml`

Reply:
Right, that's reasonable. Maybe more documentation for new functions while you are at it? :)